client/src/routes/profile.js

```diff
@@ -186,13 +180,11 @@ const TeamCodeCard = withStyles({
       <div class='content'>
         <p>Team Code</p>
         <p class='font-thin'>Copy this code and store it in a safe place as it is required to login. Then, share it with your teammates so that they can <a href='/login'>login</a> too!</p>
-        <blockquote class={classes.quote} onClick={onTeamCodeClick}>
-          {teamToken}
-        </blockquote>
+        <TokenPreview token={teamToken} onClick={onTeamCodeClick} />
```

Comment:
Do we want to put the "click to copy" functionality in the TokenPreview component as well?